### ISSUE TITLE
Accessability Improvements

### ISSUE THREAD

**Issue by AndreBastos23** [Aidurber/react-picky#37]
Hello again Aidurber,

Why does the aria-owns attribute have a invalid value? Is there a reason for it to be empty? And if you have a aria-role attribute in the parent, you should have them on the childrens, according to a colleague of mine. Is there any way for us to fix this?

Regards,
André Bastos

**Comment by Aidurber** [comment]
Hello again,
My apologies, I know very little WAI-ARIA.
Currently there is no way to override that.
I'll take a look either tonight or tomorrow night. If you and your colleague can think of a quick fix, I'd gladly accept a PR.

Thanks for reporting this issue.

**Comment by krishna63** [comment]
@AndreBastos23 : So removing this [condition ](https://github.com/Aidurber/react-picky/blob/595b07eb82491046aec1c4a27004707dce900b4d/src/Picky.tsx#L727)should solve the problem, right ?

cc: @Aidurber

**Comment by Aidurber** [comment]
@krishna63 I'm really not sure, I'm shamefully poor at accessibility. Do you think it would help?
